scss/_patterns_grid.scss

```diff
@@ -180,3 +157,204 @@
     }
   }
 }
+
+@mixin vf-l-fluid-breakout(
+  $l-fluid-breakout-aside-width: $l-fluid-breakout-aside-width,
+  $l-fluid-breakout-main-child-width: $l-fluid-breakout-main-child-width,
+  $grid-gap: $sp-unit * 2,
+  $suffix: ''
+) {
+  %l-fluid-breakout-ie11-fallback {
+    display: flex;
+
+    @media (max-width: $threshold-4-6-col) {
+      flex-direction: column;
+    }
+  }
+
+  %l-fluid-breakout__full-and-main-ie11-fallback {
```

Comment:
As it seems that we want to allow including `vf-l-fluid-breakout` multiple times in same SCSS with different params, defining %placeholders inside of it may cause issues. Especially that these placeholders seem to use the parameters of the mixin (`$l-fluid-breakout-aside-width`, `$l-fluid-breakout-main-child-width`), because these placeholders will all be added to the same global scope under the same name.

Not sure if that's supported in SCSS, but worth trying, to do the same with placeholder name as with class name `.l-fluid-breakout#{$suffix}` so placeholder would be `%l-fluid-breakout__full-and-main-ie11-fallback#{suffix}` (but I have no idea if that's possible).

Reply:
seems to work fine, nothing breaks